src/components/TodoList/TodoList.tsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import { TransitionGroup, CSSTransition } from 'react-transition-group';
+import { Error, Todo } from '../../types/todo';
+import { TodoItem } from '../TodoItem/TodoItem';
+
+type Props = {
+  todos: Todo[];
+  setTodos: (todos: Todo[]) => void;
+  setHasError: (value: Error) => void;
+};
+
+export const TodoList: React.FC<Props> = ({
+  todos,
+  setTodos,
+  setHasError,
+}) => {
+  return (
+    <section className="todoapp__main" data-cy="TodoList">
+      <TransitionGroup>
+        {todos.map(todo => (
+          <CSSTransition
+            key={todo.id}
+            timeout={300}
+            classNames="item"
+          >
+            <TodoItem
```

Comment:
Task was to add placeholder item to make adding process more smooth
Also, you need to add animations to scss, so transitions can work correctly